### DISCUSSION TITLE
Project Management

### DISCUSSION THREAD

**Discussion by seekanddefine** [knowledge-for-good/knowledgeforgood#29] [General]
Home of all things pm.

**Comment by seekanddefine** [comment]
Do we adopt Agile (iteration philosophy)/Waterfall/Lean-Six Sigma as a project management philosophy? What other methodologies (CMP, XP, Scrum, PRINCE2,?) should be considered?

Q1. What will help us create effective project management plans?

Q2: What will help us identify and manage risks?

Q3: What will help us stay within constraints?

**Reply by avidrucker** [reply]
I'd like to look into what other tech non-for-profits / co-ops have leveraged, to see if we can get an idea of what would be beneficial to us.

**Reply by avidrucker** [reply]
I'd be down also to brainstorm on any of these three questions. For a moment in mind, trying to consider all three all at once, reminds me of fording down the river in Oregon Trail (2?) https://theandrewbailey.com/fileImmutable/AXXszUbt/post/2020_OregonTrailRaft.png

**Reply by seekanddefine** [reply]
See #86
